addons/knobs/package.json

```diff
@@ -26,9 +26,11 @@
     "prop-types": "^15.5.8",
     "react-color": "^2.11.4",
     "react-datetime": "^2.8.10",
-    "react-textarea-autosize": "^4.3.0"
+    "react-textarea-autosize": "^4.3.0",
+    "util-deprecate": "1.0.2"
   },
   "devDependencies": {
+    "vue": "2.3.4",
```

Comment:
No, just that `vue` is included when developing the package (i.e. running the tests).